AppCenterDistribute/AppCenterDistributeTests/Util/MSMockReachabilityUtil.m

```diff
@@ -0,0 +1,35 @@
+#import "MSMockReachabilityUtil.h"
+#import "MSTestFrameworks.h"
+
+static NSString *kMSNetworkReachabilityChangedNotificationName = @"kMSNetworkReachabilityChangedNotification";
+
+@interface MSMockReachabilityUtil ()
+
+@property(nonatomic) id mockReachabilityUtil;
```

Comment:
Remove it and just return mock from `mockReachability`

Reply:
I think he needs it to do the stop mocking.

Reply:
changed